test/lint/lint-python.py

```diff
@@ -78,28 +78,20 @@
     'F632,'  # use ==/!= to compare str, bytes, and int literals
     'F701,'  # a break statement outside of a while or for loop
     'F702,'  # a continue statement outside of a while or for loop
-    'F703,'  # a continue statement in a finally block in a loop
     'F704,'  # a yield or yield from statement outside of a function
-    'F705,'  # a return statement with arguments inside a generator
     'F706,'  # a return statement outside of a function/method
```

Comment:
In commit: `lint: Remove python lint rules that are SyntaxError`

Nit: I believe all of F701-F707 result in `SyntaxError` and should be removed

<details>

<summary> The following examples resulted in SyntaxError's for Python 2.7.18 </summary>

f701.py:
```
def func(x):
    if x > 5:
        break

for i in range(10):
    func(i)        # <--- call function
```

f702.py:
```
def func(x):
    if x > 5:
        continue

for i in range(10):
    func(i)
```

f704.py:
```
step = 1

for x in range(0, 10, step):
  for y in range(0, 10, step):
    yield (x,y)
```

f706.py:
```
step = 1

for x in range(0, 10, step):
  for y in range(0, 10, step):
    return (x,y)
```

f707.py:
```
try:
    x = 1 / 0
except ZeroDivisionError:
    print("Divided by zero")
except:
    print("Generic exception")
except ValueError:
    print("Value error")
```

</details>